File: problem_191/sol.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        # too slow
        """
        sum = 0
        for num in bin(n)[2::]:
            sum += int(num)
        return sum
        """
        # a little slow
        return bin(n).count('1')

File: 171_Excel_Sheet_Column_Number/sol.cpp
class Solution {
    public:
        int titleToNumber(string s) {
            int num = 0;
            for(int i=0;i<s.size();i++)
            {    
                num *= 26;
                num += s[i] - 64;
            }
            return num;
        }
};
